src/components/transport_manager/test/transport_manager_default_test.cc

```diff
@@ -66,6 +65,7 @@ TEST(TestTransportManagerDefault, DISABLED_Init_LastStateUsed) {
   EXPECT_CALL(transport_manager_settings, transport_manager_tcp_adapter_port())
       .WillRepeatedly(Return(1u));
   transport_manager.Init(last_state);
+  usleep(3000);
```

Comment:
@VProdanov in this case you could use conditional variable for example to wait thread. Could you?